sim/build.sbt

```diff
@@ -12,6 +12,11 @@ lazy val commonSettings = Seq(
   libraryDependencies += "org.json4s" %% "json4s-native" % "3.6.1",
   libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
   addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
+  // ScalaDoc
+  autoAPIMappings  := true,
+  apiMappings += (
+    (chipyardRoot / unmanagedBase).value / "firrtl.jar" -> url("https://www.chisel-lang.org/api/firrtl/latest/")
```

Comment:
Magic

Reply:
I think linking to latest is fine. There are going to be some mismatches, but any alternative is going to be a lot more work for basically no gain. People who rely on the links hopefully aren't looking at unstable stuff.

Reply:
What i'm going to do is manually update this or pull from maven when i do the release so tagged versions will be correct.